app/assets/javascripts/repositories/repository_datatable.js

```diff
@@ -865,8 +864,6 @@ var RepositoryDatatable = (function(global) {
           clearTimeout(resizeTimeout);
           resizeTimeout = setTimeout(restoreColumnSizes, 200);
         });
-
-        restoreColumnSizes();
```

Comment:
Removed since it will be called in stateSaveCallback after the datatable has been initialized